.github/workflows/nightly.yml

```diff
@@ -4,6 +4,12 @@ name: Nightly
 # This job is run at 00:00 UTC every day or on demand.
 on:
   workflow_dispatch:
+    inputs:
+      cov_push_tarball:
+          description: 'Send Coverity tarball'
+          required: true
+          default: 'true'
+          type: boolean
   schedule:
     - cron:  '0 0 * * *'
 
```

Comment:
Yes, it was discussed with @bratpiorka

Reply:
yes, it makes things simpler because we needn't to pass any configuration got from UI. Also, if we trigger the Coverity manually we don't want to run Valgrind and other nightly checks.

Reply:
hmm I think that doesn't stand in conflict.. but it's ok. We can have it only in the sep. workflow, but please remove then these new params/inputs for coverity in this file 😉